source/extensions/common/wasm/wasm.cc

```diff
@@ -1025,6 +1025,11 @@ void Wasm::establishEnvironment() {
 }
 
 void Wasm::getFunctions() {
+#define _GET(_fn) getFunction(wasm_vm_.get(), #_fn, &_fn##_);
+  _GET(_malloc);
+  _GET(_free);
```

Comment:
Nit: this could be:
```
#define _GET(_fn) getFunction(wasm_vm_.get(), "_" #_fn, &_fn##_);
  _GET(malloc);
  _GET(free);
```
to match style of `_GET_PROXY`.

Reply:
Done